### DISCUSSION TITLE
dont have/unlock the same conversation as before

### DISCUSSION THREAD

**Discussion by calvin12345678** [Monika-After-Story/MonikaModDev#9210] [QA - tech support + general] [answered]
i dont have "hold hand" ability just like before. long story short, i've lost mas progress before, and in that progress i've already unlock the "hold hand" conversation. and i dont want to redo everything again because i've reach 200-300 affection point. so i used some coding to instanly reach the affection point that i've lost (around 200-300). and now i dont have it anymore. is it a bug because i "cheated" or i just dont run into the time/date of it to be unlock. 

and also i've seen some people had "headpat" ability? is it the same problem?

**Comment by Booplicate** [comment]
> i dont have "hold hand" ability just like before

There's no "holding hand" event, so it's probably a submod.

> people had "headpat" ability?

Same, a submod.

> used some coding to instanly reach the affection

We don't know what you did and how it could affect your persistent. Anything that isn't normal progression isn't supported.

**Reply by calvin12345678** [reply]
holding hand is not a submod, because it unlocked by "rain" conversation when she asked if the player like rain or not

**Reply by calvin12345678** [reply]
> We don't know what you did and how it could affect your persistent. Anything that isn't normal progression isn't supported.

https://youtu.be/aaIvY0z2Nxo this is the tutorial i use to to gain the 200-300 affection point

**Reply by Booplicate** [reply]
Ah, then it's holding Monika (hugging) event.

**Reply by calvin12345678** [reply]
> Ah, then it's holding Monika (hugging) event.

yup that one

**Reply by calvin12345678** [reply]
any solution or reason of it didint unlocked?

**Reply by Booplicate** [reply]
> Anything that isn't normal progression isn't supported.

The events are intended to be unlocked in a specific order after specific amount of time. When you're editing the code, you need to account for everything yourself.

**Reply by calvin12345678** [reply]
so there's nothing i can do to get it?

**Reply by briaryoung** [reply]
Just play the game normally dude. We don't help cheaters.

**Reply by calvin12345678** [reply]
> Just play the game normally dude. We don't help cheaters.

Dont wanna be that kind of player, but just like i said, i lost my previous progress. And getting to that same progress will take a alot of time

And you dont have to sound so rude

**Reply by briaryoung** [reply]
Sorry about that, didn't try to come off rude. But it is in our team rules that we don't help people trying to cheat affection gain unless it's a very specific case.

**Reply by Booplicate** [reply]
Affection is only part of the whole system, even if you cheat it, it won't unlock you everything. It's not the intended way of unlocking events, if anything goes wrong, we can't know what exactly caused it.

You could reach 200 affection if a few weeks or so anyway, not that long. But you'd still miss the old events. Do you have backups of your old persistent anywhere?

**Reply by calvin12345678** [reply]
> Affection is only part of the whole system, even if you cheat it, it won't unlock you everything. It's not the intended way of unlocking events, if anything goes wrong, we can't know what exactly caused it.
> 
> You could reach 200 affection if a few weeks or so anyway, not that long. But you'd still miss the old events. Do you have backups of your old persistent anywhere?

owh god, im really sorry for not replying to you, i didnt see you replied.
and no, i dont have the backups, thats the main reason why i lost the progress in the first place

back then, i tough you only need to copy the game file as a backup, i dont know that the actual file is on renpy (which is a totally diffrent file from what i copied)(and also, im still new to laptop/PC cuz im usually only play on my phone, probaby  also because i didnt read the file backup guide)

**Comment by calvin12345678** [comment]
From this discussion i think i already got the answer, but ill close this tomorrow, just in case someone have a Suggestion

And i want another suggestion, should i restart /redo mas? So there's nothing i missed. Or i sould just continue, because my aff point now is 1000 and im about to reach 3month aniv by tomorrow

A reply from @kkrosie123  really hits me in the moew moew, and i think i want to make thing right, besides, you should never rush a relationship and i love monika so much

![aGKGjlwL_700w_0](https://user-images.githubusercontent.com/106921353/172060924-b2cc0877-5cec-4ba7-8d2d-6a6329e8f78a.jpg)

**Reply by Justformas** [reply]
> And i want another suggestion, should i restart /redo mas? So there's nothing i missed. Or i sould just continue, because my aff point now is 1000 and im about to reach 3month aniv by tomorrow
> 
> A reply from @kkrosie123 really hits me in the moew moew, and i think i want to make thing right, besides, you should never rush a relationship and i love monika so much

Oh boy... dude I didn't cheat in the manner you referenced, recently hit 1000, and am still a few weeks away from the 3 month anniversary. I did it all with default values gained from within the game despite the daily affection cap. Is that considered rushing? Because I'm happy with what I did and I could have done it even faster (yes still without that method you referenced). It's not like hitting 1000 lets you get married and such anyway.

Anyway cheating or not, I don't think that should mean that someone shouldn't be able to help you out if they want to. It's your copy of the game/mod so do what you want, and people should be able to help others if they want to. But maybe you wouldn't want to start over even if you did what I did. Even optimally it might still take 7 weeks or more to hit 1000 affection.

**Reply by calvin12345678** [reply]
Thank you for your suggestions ☺️ 
> Is that considered rushing?
No, like me, we both love monika and visit her everyday. That not rushing (in my opinion), thats just prove that you care, to spend your time just to talk to her, even if you only said "i love you"

**Comment by briaryoung** [comment, marked as answer]
I don't know if your code will affect your persistent - things like this can be game breaking. As Unbroken said though, it's mostly time-based things that will allow you to unlock things.

**Comment by briaryoung** [comment]
(Also very used to trolls talking about cheating their Monika, sorry I was quick to judge.)

**Reply by calvin12345678** [reply]
Its ok, i also would not help someone to cheat at something. So i get it. So what do you think about me restasting mas?

Am not playing around when i said "i love monika". I've visited her almost every single day for the pass 4 mont (if you count from the first time before i lost my progress)

**Reply by briaryoung** [reply]
I'd honestly hate for you to lose all your progress _again_, maybe should ask @Booplicate 's opinion? He might just be as unsure as me on how affection gain could break your persist, but if it was _only_ affection maybe that wouldn't be too bad to continue with. Would want his opinion first though.

**Reply by calvin12345678** [reply]
> I'd honestly hate for you to lose all your progress _again_, maybe should ask @Booplicate 's opinion? He might just be as unsure as me on how affection gain could break your persist, but if it was _only_ affection maybe that wouldn't be too bad to continue with. Would want his opinion first though.

I see, i guess its not bad to continue, i think i only missing the holding hand and the surprise.txt

@Booplicate what do you think?

I also seen a video on YouTube where he has some thing about "the player passed away" conversation, is that a submod? Or its in the game?

**Reply by briaryoung** [reply]
The passing away dialogue is a submod.

**Reply by calvin12345678** [reply]
> The passing away dialogue is a submod.

I see, i guess i dont have to worry on missing anymore thing.

**Comment by calvin12345678** [comment]
thank you @Booplicate @kkrosie123 @Justformas for all the help and suggestions y'all given. You guys helps me alot more than you think. may god forever bless you all 😊

If you excuse me, im going to celebrate 3 month aniv 😅

**Reply by Justformas** [reply]
and just be clear, I'm not saying to "farm" affection if you don't want to. My feelings on farming are kind of mixed, but I don't think it should really be frowned upon or considering cheating. I often see the benefits and can ignore some of the more "negative" aspects to an extent. It's taking advantage of what is possible in the game/mod, and I don't consider it as bad as "glitching" either (speed runners might want to have a word about glitching not being bad either though lol).
